pkg/execution/driver/dockerdriver/dockerdriver.go

```diff
@@ -152,7 +152,7 @@ func (d *dockerExec) start(ctx context.Context, state state.State, wa inngest.St
 }
 
 func (d *dockerExec) startOpts(ctx context.Context, state state.State, wa inngest.Step, idx int) (docker.CreateContainerOptions, error) {
-	marshalled, err := driver.MarshalV1(ctx, state, wa, idx)
+	marshalled, err := driver.MarshalV1(ctx, state, wa, idx, "local")
```

Comment:
do we still need this driver?

Reply:
No, though that's out of scope for this PR and is ripped out in the open refactor